Python Directory (#8)
ubccr/ccr-examples: slurm/2_ApplicationSpecific/python/README.md

# Python on the CCR Clusters

> This directory includes examples of serial, multithreaded, and GPU  Python jobs. (CCR Staff: Need more info here)

Reviewer: We will want to modify this eventually. I would put something like this suggestion, but since we plan on adding those others soon it's fine as is, we would have to change it back soon anyway.
```suggestion
This directory includes examples of a serial Python job, with multithreaded and GPU examples coming soon.
```

Reviewer 2: @andreakyaw I'm not seeing that this suggestion from Kevin made it into your updates.

## Placeholders:

Portions of the example batch scripts contain placeholders, denoted by square brackets. Refer to the 2_ApplicationSpecific 

> [README](../README.md) for details and options.

Reviewer: I would not have this be a new line. Combine line 7-8
```suggestion
Portions of the example batch scripts contain placeholders, denoted by square brackets. Refer to the 2_ApplicationSpecific [README](../README.md) for details and options.
```

## Serial Python job ([serial/](./serial))

A serial Python job is one that requires only a single CPU-core.

> Provided is an example of a trivial, one-line serial Python program (`fibonacci.py`) with the corresponding Slurm script (`python-sp.sh`) that can be modified to run a serial Python job.

Reviewer: Technically not a one line python script. I also wouldn't call it trivial, that will make people who know what they're doing want to skip it and make people who don't think it's trivial feel bad if they struggle with it.

Reviewer 2: @andreakyaw please also resolve Kevin's concerns with this section.

To run the Python script, simply submit the job to the scheduler from a login node with the following command:
```

> $ sbatch python-sp.sh

Reviewer: This is fine, just leaving a note for the contributors at large about how we want to format terminal commands, e.g., with or without $. I think the consensus was to use $ for terminal and > for python interpreter. (I think that maybe should also apply to MATLAB).
